src/main/java/org/folio/oaipmh/helpers/ItemsStorageHelper.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.oaipmh.helpers;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+import java.time.Instant;
+
+public interface ItemsStorageHelper {
```

Comment:
The reason why I decided to name interface as 'Items' because according to [2.3 Item](https://www.openarchives.org/OAI/openarchivesprotocol.html#Item) section of the OAI-PMH spec:

> An _item_ is a constituent of a repository from which metadata about a resource can be disseminated. An item is conceptually a container that stores or dynamically generates metadata about a single resource in [multiple formats](https://www.openarchives.org/OAI/openarchivesprotocol.html#MetadataNamespaces), each of which can be harvested as [records](https://www.openarchives.org/OAI/openarchivesprotocol.html#Record) via the OAI-PMH. Each item has an [identifier](https://www.openarchives.org/OAI/openarchivesprotocol.html#UniqueIdentifier) that is unique within the scope of the repository of which it is a constituent.

**mod-inventory-storage** defines [instance-storage.raml](https://github.com/folio-org/mod-inventory-storage/blob/master/ramls/instance-storage.raml) which describes endpoints to work with instances. However **mod-source-record-storage** defines [source-record-storage.raml](https://github.com/folio-org/mod-source-record-storage/blob/master/ramls/source-record-storage.raml) which contains `/record-storage/items` endpoint and looks like works with items.

Anyway we can rename the interface to `RecordsStorageHelper` which will have `items` and `records` related methods...